lib/openstates/models/bill.rb

```diff
@@ -48,6 +48,12 @@ def bill_details(state, session, bill_id)
 
         from_hash(response)
       end
+
+      def find_by_os_id(os_id)
```

Comment:
I know it's a bit more wordy, but how about `find_by_open_state_id`?

I much prefer to avoid semi ambiguous method names and the `os` in this method name makes me think of operating system (clearly has nothing to do with legislative bills and such 😊). I've found that super specific method names help to avoid confusion and keep the code self documented.